functors task (1)/student/include/impl.h

```diff
@@ -1,13 +1,22 @@
 #pragma once
 #include "bar_serving.h"
 #include <functional>
+#include <iostream>
 
 /**
  * @todo Implement functor-generator that will return next beer brand (cyclic)
  * @example HoeGaarden -> Corona -> Carlsberg -> Bud -> ZlataPraha -> Leffe -> HoeGaarden...
  */
 struct BeerOrganizer
 {
+    size_t state {static_cast<size_t>(BeerBrand::None)};
```

Comment:
_consider more descriptive name_
why not BeerBrand type?

Reply:
Done